### PR TITLE
Address search results

### PR DESCRIPTION
**Pull request by bogdan-zaharia**
- Added results page for searching by address
- Added AddressResults component for the table
- Added Dropbox component for the dropdown list

### CODE REVIEW

File: tmp/req.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (bjpirt) [inline tmp/req.json:1]: Should this be in the pr?

File: tmp/response.json [hunk 2]
```diff
@@ -0,0 +1,296 @@
+{
```
Comment (bjpirt) [inline tmp/response.json:1]: Should this be in the pr?

File: tmp/req.json (hunk 1 above)
Comment (kosiakkatrina) [reply tmp/req.json]: No! My bad. Should be gone now

File: cypress/fixtures/searchAddress.json [hunk 3]
```diff
@@ -0,0 +1,222 @@
+{
```
Comment (bjpirt) [inline cypress/fixtures/searchAddress.json:1]: Are these anonymised?

**Review by bjpirt — APPROVED** [review]

**Pull request merged**